device-types/Juniper/EX4400-48F.yaml

```diff
@@ -0,0 +1,124 @@
+---
+manufacturer: Juniper
+model: EX4400-48F
+slug: juniper-ex4400-48f
+part_number: EX4400-48F
+u_height: 1
+is_full_depth: true
+comments: '[Juniper EX4400 Hardware Guide](https://www.juniper.net/documentation/us/en/hardware/ex4400/topics/topic-map/ex4400-system-overview.html)'
+interfaces:
+  - name: me0
+    type: 1000base-t
+    mgmt_only: true
+  - name: ge-0/0/0
+    type: 1000base-x-sfp
+  - name: ge-0/0/1
+    type: 1000base-x-sfp
+  - name: ge-0/0/2
+    type: 1000base-x-sfp
+  - name: ge-0/0/3
+    type: 1000base-x-sfp
+  - name: ge-0/0/4
+    type: 1000base-x-sfp
+  - name: ge-0/0/5
+    type: 1000base-x-sfp
+  - name: ge-0/0/6
+    type: 1000base-x-sfp
+  - name: ge-0/0/7
+    type: 1000base-x-sfp
+  - name: ge-0/0/8
+    type: 1000base-x-sfp
+  - name: ge-0/0/9
+    type: 1000base-x-sfp
+  - name: ge-0/0/10
+    type: 1000base-x-sfp
+  - name: ge-0/0/11
+    type: 1000base-x-sfp
+  - name: ge-0/0/12
+    type: 1000base-x-sfp
+  - name: ge-0/0/13
+    type: 1000base-x-sfp
+  - name: ge-0/0/14
+    type: 1000base-x-sfp
+  - name: ge-0/0/15
+    type: 1000base-x-sfp
+  - name: ge-0/0/16
+    type: 1000base-x-sfp
+  - name: ge-0/0/17
+    type: 1000base-x-sfp
+  - name: ge-0/0/18
+    type: 1000base-x-sfp
+  - name: ge-0/0/19
+    type: 1000base-x-sfp
+  - name: ge-0/0/20
+    type: 1000base-x-sfp
+  - name: ge-0/0/21
+    type: 1000base-x-sfp
+  - name: ge-0/0/22
+    type: 1000base-x-sfp
+  - name: ge-0/0/23
+    type: 1000base-x-sfp
+  - name: ge-0/0/24
+    type: 1000base-x-sfp
+  - name: ge-0/0/25
+    type: 1000base-x-sfp
+  - name: ge-0/0/26
+    type: 1000base-x-sfp
+  - name: ge-0/0/27
+    type: 1000base-x-sfp
+  - name: ge-0/0/28
+    type: 1000base-x-sfp
+  - name: ge-0/0/29
+    type: 1000base-x-sfp
+  - name: ge-0/0/30
+    type: 1000base-x-sfp
+  - name: ge-0/0/31
+    type: 1000base-x-sfp
+  - name: ge-0/0/32
+    type: 1000base-x-sfp
+  - name: ge-0/0/33
+    type: 1000base-x-sfp
+  - name: ge-0/0/34
+    type: 1000base-x-sfp
+  - name: ge-0/0/35
+    type: 1000base-x-sfp
+  - name: xe-0/0/36
+    type: 10gbase-x-sfpp
+  - name: xe-0/0/37
+    type: 10gbase-x-sfpp
+  - name: xe-0/0/38
+    type: 10gbase-x-sfpp
+  - name: xe-0/0/39
+    type: 10gbase-x-sfpp
+  - name: xe-0/0/40
+    type: 10gbase-x-sfpp
+  - name: xe-0/0/41
+    type: 10gbase-x-sfpp
+  - name: xe-0/0/42
+    type: 10gbase-x-sfpp
+  - name: xe-0/0/43
+    type: 10gbase-x-sfpp
+  - name: xe-0/0/44
+    type: 10gbase-x-sfpp
+  - name: xe-0/0/45
+    type: 10gbase-x-sfpp
+  - name: xe-0/0/46
+    type: 10gbase-x-sfpp
+  - name: xe-0/0/47
+    type: 10gbase-x-sfpp
+  - name: et-0/1/0
+    type: 100gbase-x-qsfp28
+  - name: et-0/1/1
+    type: 100gbase-x-qsfp28
+console-ports:
+  - name: Console
+    type: rj-45
+  - name: Front Console
+    type: usb-c
+power-ports:
+  - name: PSU0
```

Comment:
It looks like this devices power supplies are modular. If this is the case, then the PSUs should actually be `module-bays` and the applicable PSU units should be created and added to the module-types folder